connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java

```diff
@@ -150,13 +158,13 @@ public void testShowTables() throws ApiException, IOException {
     setupExternalParquetTable(PARQUET_TABLE, new ArrayList<>(0));
 
     Row[] tables = (Row[]) session.sql("SHOW TABLES in " + SCHEMA_NAME).collect();
-    assertEquals(tables.length, 1);
-    assertEquals(tables[0].getString(0), SCHEMA_NAME);
-    assertEquals(tables[0].getString(1), PARQUET_TABLE);
+    assertThat(1).isEqualTo(tables.length);
+    assertThat(SCHEMA_NAME).isEqualTo(tables[0].getString(0));
```

Comment:
the actual/expected is the wrong way around here and below. Should be `assertThat(tables[0].getString(0)).isEqualTo(SCHEMA_NAME)`